input/ignoreWarnings.txt

```diff
@@ -1,6 +1,7 @@
 == Suppressed Messages ==
 
-# SUPPRESS US CORE OBSERVATION MESSAGES. This was approved by Cross-Group Projects Work Group on 1/28/2021 Floyd Eisenberg/Brett Marquard 28-0-1. See https://jira.hl7.org/browse/FHIR-30707
+# SUPPRESS US CORE OBSERVATION MESSAGES. For STU2 This was approved by Cross-Group Projects Work Group on 1/28/2021 Floyd Eisenberg/Brett Marquard 28-0-1. See https://jira.hl7.org/browse/FHIR-30707
+# For STU3, this was approved by the Cross-Group Projects Work Group on 2/9/2023   Saul Kravitz/Floyd Eisenberg: 23-0-2    see https://jira.hl7.org/browse/FHIR-40383
```

Comment:
Adding the vote is good, but you also have to add the WARNING: messages to suppress from the QA page